Makefile

```diff
@@ -106,6 +110,19 @@ download-illumos-gate: FRC
 download-u-boot: $(SRCS)
 	git clone --shallow-since=2019-01-01 -b v2022.10 \
 	    https://github.com/u-boot/u-boot $(SRCS)/u-boot
+	cd $(SRCS)/u-boot && patch -p1 < $(PWD)/patches/u-boot.patch
+
+download-arm-trusted-firmware: $(SRCS)
+	  git clone --depth=1 --branch v2.9.0 \
+	      https://github.com/ARM-software/arm-trusted-firmware \
```

Comment:
Does this mean I can delete my fork?

Reply:
Yes! I reverted to using stock here, and for u-boot (and applying a patch to u-boot to set the boot option macros).